src/briefcase/integrations/linuxdeploy.py

```diff
@@ -77,3 +78,49 @@ def upgrade(self):
             print("...done.")
         else:
             raise MissingToolError('linuxdeploy')
+
+    def patch_elf_header(self):
+        """
+        Patch the ELF header of the AppImage to ensure it's always executable.
+
+        This patch is necessary on Linux hosts that use AppImageLauncher.
+        AppImages use a modified ELF binary header starting at offset 0x08
+        for additional identification. If a system has AppImageLauncher,
+        the Linux kernel module `binfmt-misc` will try to load the AppImage
+        with AppImageLauncher. As this binary does not exist in the Docker
+        container context, we patch the ELF header of linuxdeploy to remove
+        the AppImage bits, thus making all systems treat it like a regular
+        ELF binary.
+
+        Citations:
+        - https://github.com/AppImage/AppImageKit/issues/1027#issuecomment-1028232809
+        - https://github.com/AppImage/AppImageKit/issues/828
+        """
+
+        patch = {
+            'offset': 0x08,
+            'original': bytes.fromhex('414902'),
+            'patch': bytes.fromhex('000000')
+        }
```

Comment:
This is structured as a dictionary, but there isn't really any reason for it to be; they're effectively constants. We're not doing anything dynamic here, so we're not gaining anything from storing these as a dictionary; and a dictionary lookup will be *very* marginally slower, because there's an extra key lookup operation. 

I'd even argue they should be module-level constants - they're "fundamental" properties of the ELF binary, and they're duplicated in the test, rather than referring to these fundamental properties.

Reply:
Makes sense to me! I've changed those to constants, and am using those in the tests as well.